sigma9.css

```diff
@@ -1,3 +1,10 @@
+@charset "utf-8";
+ 
+@import url('http://www.scp-wiki.net/local--files/component:theme/font-bauhaus.css');
+@import url('http://fonts.googleapis.com/earlyaccess/nanumgothic.css');
+@import url('http://fonts.googleapis.com/earlyaccess/nanumgothiccoding.css');
+@import url('http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css');
+
```

Comment:
Nitpick, I would have the informational comment be before the imports. Since it's a comment it won't affect CSS behavior.